cmds/dhclient/dhclient.go

```diff
@@ -37,6 +37,7 @@ const (
 
 var (
 	leasetimeout = flag.Int("timeout", 15, "Lease timeout in seconds")
+	retry        = flag.Int("retry", -1, "Max number of attempts to send DHCPv6 solicit. -1 means infinity")
```

Comment:
Maybe we can do this retry with dhcp4, too? There's a networkError.Timeout case below in dhclient4 where you could maybe just retry?